### DISCUSSION TITLE
live weather animation

### DISCUSSION THREAD

**Discussion by ShaggyDaddyDan** [rocksdanister/lively#1985] [Ideas]
would it be possible to have a animated wallpaper showing the live weather in whatever city the user chooses?

**Comment by rocksdanister** [comment]
You mean like weather animation for each weather?

**Reply by ShaggyDaddyDan** [reply]
yea but it would be live from the users choice of area

**Comment by mohrpw** [comment]
What about using https://github.com/chubin/wttr.in for this?

**Reply by rocksdanister** [reply]
I already have a client implementation:
https://github.com/rocksdanister/weather

**Comment by rocksdanister** [comment]
closing as dup https://github.com/rocksdanister/lively/discussions/2087
